app/relisten/(tabs)/artists/[artistUuid]/songs.tsx

```diff
@@ -25,7 +30,7 @@ export default function Page() {
       <DisappearingHeaderScreen
         ScrollableComponent={SongList}
         songs={Array.from(data.songs)}
-        filterPersistenceKey={['artists', artistUuid, 'songs'].join('/')}
+        filterOptions={{ persistence: { key: ['artists', artistUuid, 'songs'].join('/') } }}
```

Comment:
maybe just let persistence.key accept an array and stringify it if its an array, instead of repeating `.join` everywhere

Reply:
good idea. i'll refactor that later